webroot/src/pages/AuthCallback/AuthCallback.ts

```diff
@@ -39,15 +39,46 @@ export default class AuthCallback extends Vue {
         return this.$route.query?.code?.toString() || '';
     }
 
+    get userType(): string {
+        return this.$route.query?.state?.toString() || '';
```

Comment:
"State" is a specific query param that cognito uses to pass information back and forth during the login redirects. The value put in state when we visit the login screen is then persisted / passed back to us in the post login redirect url.

In this case we are using it keep track of which login screen we are returning from so that we can make only the correct API call to get the Auth tokens.

Reply:
Got it. Can we add this somewhere as a comment, perhaps with a link to AWS docs? (https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html)